docs/content/en/integrations/parsers/file/mend.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Mend Scan"
+toc_hide: true
+---
+
+### File Types
+Accepts a JSON file, generated from the Mend* Unified Agent.  
+
+### Sample Scan Data / Unit Tests
+Unit tests for Mend JSON files can be found at https://github.com/DefectDojo/django-DefectDojo/blob/76e11c21e88fb84b67b6da27c78fbbe1899e7e78/unittests/scans/whitesource/cli_generated_many_vulns.json
```

Comment:
Since we're handling the parser/unit test changes in another PR, this link won't work until that's merged, but it should be the correct link in the future once both of these PRs are merged into `master`.

```suggestion
Unit tests for Mend JSON files can be found at https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/mend
```